src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +211,21 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        startUserCommandFeedbackLoop();
+    }
+
+	/**
+	 * Starts a user input, console output loop for the user
+	 * T2A3: Reduce SLAP of Method
```

Comment:
W.r.t. the reference to `T2A3`, the comment should be strictly about the code. You can use github comments (like this one) to communicate with the reviewer/tutor.